resources/tests/build_kernel.sh

```diff
@@ -90,7 +90,9 @@ build_kernel() {
     cp $binary_path $binary_name
 
     popd >/dev/null
-    echo "Kernel binary placed in build/kernel/$binary_name"
+
+    local full_binary_name="$KERNEL_DEST_DIR/$binary_name"
```

Comment:
super-nitpick: `full_binary_name` -> `full_binary_path`

Reply:
You're right, this looks more consistent. Fixed, thanks.